src/psyclone/tests/parse/algorithm_test.py

```diff
@@ -38,6 +38,7 @@
 tests for code that is not covered there.'''
 
 from __future__ import absolute_import
+import os
```

Comment:
I am afraid this test module only works when the `pytest` is run from the `src/psyclone/tests/parse` directory where the file is located. It should work regardless of the location when running `pytest`.

Reply:
I confirm that I also get the same error - oops.

Reply:
All working now, thanks for fixing 😄